### PR TITLE
[RW-6031] Change "All" to "Any" fitbit data

### PR DESCRIPTION
**Pull request by mhigbyflowers**
Change "All" to "Any" for Fitbit data description and modify the sentence

### CODE REVIEW

File: public-api/db-cdr/generate-cdr/run-measurement-queries.sh [hunk 1]
```diff
@@ -1888,11 +1888,11 @@ ceil(TIMESTAMP_DIFF(cast(datetime as timestamp), birth_datetime, DAY)/365.25) as
      select co.person_id, p.gender_concept_id as gender
      from \`${BQ_PROJECT}.${BQ_DATASET}.steps_intraday\` co join \`${BQ_PROJECT}.${BQ_DATASET}.person\` p on p.person_id=co.person_id
      )
-     select 0 as id, 3101 as analysis_id, 'All Fitbit Data' as stratum_1, cast(gender as string) as stratum_2, 'Fitbit' as stratum_3,
+     select 0 as id, 3101 as analysis_id, 'Any Fitbit Data' as stratum_1, cast(gender as string) as stratum_2, 'Fitbit' as stratum_3,
      count(distinct ca.person_id) as count_value, count(distinct ca.person_id) as source_count_value
      from m_gender ca group by 4
      union all
-     select 0 as id, 3102 as analysis_id, 'All Fitbit Data' as stratum_1,
```
Comment (mhigbyflowers) [inline public-api/db-cdr/generate-cdr/run-measurement-queries.sh:1895]: is this change necessary for new data generation?

File: public-api/db-cdr/generate-cdr/run-measurement-queries.sh [hunk 2]
```diff
@@ -1919,4 +1919,4 @@ select 0 as id, 3107 as analysis_id, 'Heart rate (minute-level)' as stratum_1, c
...
 union all
-select 0 as id, 3107 as analysis_id, 'All Fitbit Data' as stratum_1, concat(cast(EXTRACT(YEAR from data_date) as string), '-', case when EXTRACT(MONTH from data_date) >= 1 and EXTRACT(MONTH from data_date) <= 6 then '06-30' else '12-31' end) as stratum_2, 'Fitbit' as stratum_3, count(distinct person_id) as count_value, count(distinct person_id) as source_count_value from all_fibit_data group by 4;"
\ No newline at end of file
+select 0 as id, 3107 as analysis_id, 'Any Fitbit Data' as stratum_1, concat(cast(EXTRACT(YEAR from data_date) as string), '-', case when EXTRACT(MONTH from data_date) >= 1 and EXTRACT(MONTH from data_date) <= 6 then '06-30' else '12-31' end) as stratum_2, 'Fitbit' as stratum_3, count(distinct person_id) as count_value, count(distinct person_id) as source_count_value from all_fibit_data group by 4;"
```
Comment (mhigbyflowers) [inline public-api/db-cdr/generate-cdr/run-measurement-queries.sh]: also this change too.

**Review by SrushtiGangireddy — APPROVED** [review]
Yeah this needs new data. Thanks for changing.
